src/components/WebexMeetingControl/__snapshots__/WebexMeetingControl.test.js.snap

```diff
@@ -1,37 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Webex Meeting Control component snapshot matches with active icon button 1`] = `
-<Button
-  active={false}
+<ContextTransform(Button)
```

Comment:
Is this a momentum change that affected our snapshot?

Reply:
Yes, I was trying to figure out where this changes happened but couldn't really get a good idea. Seems like there have been changes within the codebase to use React contexts